pkg/build/gobuild.go

```diff
@@ -82,26 +85,54 @@ func NewGo(options ...Option) (Interface, error) {
 	return gbo.Open()
 }
 
+// goList is a wrapper for the `go list` command to retrieve information about a package.
+func (g *gobuild) goList(importPath, projection string) (string, error) {
+	cmd := exec.Command("go", "list", "-f", "{{."+projection+"}}", importPath)
+
+	cmd.Dir = g.wd
+
+	stdout, err := cmd.Output()
+
+	if err != nil {
+		return "", err
+	}
+
+	outputs := strings.Split(string(stdout), "\n")
+
+	if len(outputs) == 0 {
```

Comment:
I think I'd pull this error handling/checking outside of `goList` and make `goList` return `([]string, error)` or rename `goList` to sound less generic.

Reply:
changed the method name to findImportPath to be more specific.